lib/agenda/create.ts

```diff
@@ -11,9 +11,9 @@ const debug = createDebugger('agenda:create');
  * @param name name of job
  * @param data data to set for job
  */
-export const create = function(this: Agenda, name: string, data: {}): Job {
+export const create = function(this: Agenda, name: string, data: any): Job {
```

Comment:
How come `any` and not `object`? (and maybe it can also be `null`?)

Reply:
yeah, I was surprised too.
The unicorn-lint-styles do not allow 'object' but allows 'any'.

In newer Typescript there is also a Record<string, type> - type, but I'm not too familiar with it, so I didn't try that one.

Reply:
Aight! Not a biggie anyway, I don't think it needs to block the PR.